src/StudyApi/Extensions/AwsSecretsConfigurationBuilderExtensions.cs

```diff
@@ -17,7 +17,7 @@ public static IWebHostBuilder AddAwsSecrets(this IWebHostBuilder hostBuilder)
         {
             return hostBuilder.ConfigureAppConfiguration((hostingContext, configBuilder) =>
             {
-                if (hostingContext.HostingEnvironment.IsDevelopment()) return;
+                if (!hostingContext.HostingEnvironment.IsEnvironment("local")) return;
```

Comment:
`local` = `Development` under .Net conventions.

We should have a shared AWS deployed environment configuration extension that is called in `Staging` and `Production`.